test/src/080-tracing/main

```diff
@@ -0,0 +1,61 @@
+cvmfs_test_name="tracing"
+
+cvmfs_run_test() {
+    logfile=$1
+    sudo sh -c "echo \"export GLITE_VERSION='util'\" > /etc/cvmfs/config.d/grid.cern.ch.local"
+    repo="grid.cern.ch"
+    echo "Setup..."
```

Comment:
Prefixing the explicit echo statements with `*** ` makes them easier to find in the output log